src/components/ui/Button/Button.tsx

```diff
@@ -1,20 +1,29 @@
 import React from "react";
-
 import cs from 'classnames';
-import styles from "./PrimaryButton.module.scss";
+import styles from "./Button.module.scss";
 
-interface PrimaryButtonProps {
-    type?: 'submit' | 'reset' | 'button';
-    onClick?: () => void;
+interface ButtonProps {
+    secondary?: boolean
+    type?: 'submit' | 'reset' | 'button'
+    onClick?: () => void
     size?: 'medium' | 'small'
 }
 
-export const PrimaryButton: React.FC<React.PropsWithChildren<PrimaryButtonProps>> = ({ children, type, onClick, size = 'medium' }) => {
+export const Button: React.FC<React.PropsWithChildren<ButtonProps>> = (
+    {
+        children,
+        secondary = false,
+        type,
+        onClick,
+        size = 'medium'
+    }
+) => {
     return (
         <button
             type={type}
             className={
                 cs(styles.button, {
+                    [styles.secondary]: secondary,
```

Comment:
uhhh, i would like to have primary as a prop

Reply:
done